background-charm-service/cast-admin.ts

```diff
@@ -43,7 +43,7 @@ const identity = await getIdentity(
 );
 
 storage.setRemoteStorage(new URL(toolshedUrl));
-setBobbyServerUrl(toolshedUrl);
```

Comment:
Goodbye `bobby` 🫡

Reply:
🖖

Reply:
bobby is how you know I still type code... but I use tab complete in other files, so a spelling issue in one place replicates throughout the codebase